calc.js

```diff
@@ -4,150 +4,217 @@ const keys = document.querySelector('#keyboard');
 const arithmeticKeys = document.querySelector('#arithmetic');
 const functionKeys = document.querySelector('#functions');
 
-keys.addEventListener( 'click' , getPress );
-arithmeticKeys.addEventListener( 'click' , getPress );
-functionKeys.addEventListener( 'click' , getPress );
+keys.addEventListener('click', getPress);
+arithmeticKeys.addEventListener('click', getPress);
+functionKeys.addEventListener('click', getPress);
 
 const obiekt = {
   value: 0,
   operator: '',
   secondValue: 0,
   state: ''
-} ;
+};
 
+const OPERATOR_PRESENT_KEY = 'operatorPresent'
+const SECONDAY_VALUE_KEY = 'secondaryValue'
 
+const OPERATIONS = {
+  DEVIDE_BY_X: '1/x',
+  CHANGE_SYMBOL: '+/-',
+  CE: 'CE',
+  C: 'C',
+  PERCENT: '%',
+}
 
+const TYPES = {
+  NUMBER: 'number',
+  SIGN: 'sign',
+  DOT: 'dot',
+  OPERATION: 'arith',
+  FUNCTION: 'funct',
+  SQRT: 'sqrt'
+}
+const BUTTON_CLASS = ['button', 'shadow']
+const NUMBER_KEYBOARD_CLASS = [...BUTTON_CLASS, TYPES.NUMBER]
```

Comment:
Czemu tutaj używasz operatora spread ? ( wiem, że spread pozwala np. rozbić string na poszczególne litery, albo zastosować metody obiektu Math na tablicy, ale nie rozumiem tego )

To ma coś wspólnego z dynamicznym generowaniem ?